src/components/activity-groups-list/activity-groups-list.tsx

```diff
@@ -41,6 +43,13 @@ export const ActivityGroupsList: FC<Props> = ({ activityGroups, handleUpdate = e
     return result;
   }, [activityGroups]);
 
+  const handleRefresh = () => {
```

Comment:
extract this handler to a separate file and reuse it here and in TokenList component